synergy-design-system/synergy-design-system
# docs: 📚 Raw token values for dark mode are drawn from light-mode instead

… elements where applicable now



# Pull Request

## 📖 Description

This PR fixes an issue that the wrong raw value for dark tokens where displayed on https://synergy-design-system.github.io/?path=/docs/tokens-colors--docs.

### 🎫 Issues

Closes #261.

## 👩‍💻 Reviewer Notes

Please have a look at the Tokens/Color story. It should now display the correct string representation on "Colors / dark". It should also copy the right values when pressing the Raw Value row.

## 📑 Test Plan

- Check https://synergy-design-system.github.io/?path=/docs/tokens-colors--docs to see the wrong behaviour.
- Check out this branch and issue `pnpm i -r && pnpm build && cd packages/docs && pnpm start`
- Have a look the Dark Section of Tokens / Colors. The raw values should now reflect the correct values for dark mode.

## ✅ DoD
